src/System Application/App/Environment Information/src/EnvInfNotificationImpl.Codeunit.al

```diff
@@ -0,0 +1,53 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace System.Environment;
+
+codeunit 3706 "Env. Inf. Notification Impl."
+{
+    Access = Internal;
+    SingleInstance = true;
```

Comment:
why single instance?

Reply:
Because the others are. Both the other impl. and facade. I guess it doesn't really matter here?